geode-benchmarks/src/main/java/org/apache/geode/benchmark/tasks/LocatorUtil.java

```diff
@@ -27,7 +29,7 @@
  */
 public class LocatorUtil {
   static String getLocatorString(TestContext context, int locatorPort) {
-    Set<InetAddress> locators = context.getHostsForRole("locator");
+    Set<InetAddress> locators = context.getHostsForRole(LOCATOR.name());
```

Comment:
we're talking to the `harness` layer so we have to convert from strong-typing to string-typing